### DISCUSSION TITLE
Use a different entity condition for each user?

### DISCUSSION THREAD

**Discussion by patienttruth** [NemesisRE/kiosk-mode#237] [Q&A] [answered]
I'm trying to lock down a couple of views to be used with Android device controls while the device is locked.

I asked over on the home assistant form and got this great suggestion to use the binary sensor of whether or not my device is locked. 

https://community.home-assistant.io/t/lockdown-android-device-control-panel/732454/2

The next stage is that I'd like for my wife to be able to view the same panels while her device is locked. Is there a way to configure with kiosk mode for this use case?

**Comment by patienttruth** [comment, marked as answer]
Am I asking for the same thing that this post and the linked post are asking for? 

https://github.com/NemesisRE/kiosk-mode/discussions/185

**Reply by elchininet** [reply]
Hi @patienttruth,
That is not possible at the moment.
And yes, it is similar to what you linked.
Regards

**Reply by patienttruth** [reply]
Thanks @elchininet!

**Comment by elchininet** [comment]
Hi @patienttruth,

[This pull request](https://github.com/NemesisRE/kiosk-mode/pull/274) contains something in which I have been working lately and it is related in some way to this discussion. Please, read the description of the pull request but you can also check [the documentation in the branch](https://github.com/NemesisRE/kiosk-mode/tree/support_for_jinja_templates).

Please, if you can test [this gist](https://gist.github.com/elchininet/9a2648ea0016d3194072712c745c5ad2) and check if you can spot errors or unwanted issues it would be helpful.

Regards

**Reply by patienttruth** [reply]
Wow! This is a very smart approach! I'm not in a position that I can test this very soon, but very grateful for the follow-up. I will come back around to it when I can. Thank you!
